core/src/i18n/en.yaml

```
@@ -906,7 +906,7 @@ generic-list:
 git-repositories:
   auth:
     basic: Basic
-    public: Public
+    public: PublicAfter a job is created, the containers are read-only.
```

```suggestion
    public: After a Job is created, the containers are read-only.
```

I removed the "Public", typo?

---

Ah wait, that's the only difference between 909 and 991. Bamboozled again.